9.4/paper.tex

```diff
@@ -232,7 +232,7 @@ \section{Overview}
   \item The new member function \texttt{find\_point\_owner\_rank()} of
         \texttt{parallel\allowbreak ::distributed::\allowbreak Tri\allowbreak angulation} allows one to find the MPI
   ranks of cells containing specified points.
-  It is communication-free and leverages the functionality of p4est (>v.2.2).
+  It is communication-free and leverages the functionality of \pfrst{} (>v.2.2).
```

Comment:
Does the `>` character render correctly? I thought it can only be used in math mode.

Reply:
It does render correctly as we use the package `inputenc` for type setting. Without the package, we would need to use `\textless` or math mode.
https://github.com/dealii/release-papers/blob/9b205e4ae2409f2b7f4a0af326217420136cd727/9.4/paper.tex#L3